.gitmodules

```diff
@@ -1,3 +1,4 @@
 [submodule "rust-vmm-ci"]
 	path = rust-vmm-ci
-	url = https://github.com/rust-vmm/rust-vmm-ci.git
+    url = https://github.com/aghecenco/rust-vmm-ci.git
```

Comment:
Do you think it makes sense to create a RFC PR for `rust-vmm-ci` as well with the contents of that branch, so we can add further comments there?

Reply:
* `rust-vmm-ci`: https://github.com/rust-vmm/rust-vmm-ci/pull/25
* `rust-vmm-container`: https://github.com/rust-vmm/rust-vmm-container/pull/28